src/commands/member_info.rs

```diff
@@ -8,21 +8,24 @@ use serenity::{
 #[command]
 #[aliases("minfo", "memberinfo")]
 async fn get_member_info(ctx: &Context, msg: &Message, mut args: Args) -> CommandResult {
-    let guild_id = msg.guild_id.unwrap();
+    let guild_id = match msg.guild_id {
```

Comment:
Since the bot has DMs disabled by default, it's better to just unwrap the guild ID since all commands are invoked from guilds.